runtimes/typescript/lbr-plutus/build.nix

```diff
@@ -0,0 +1,15 @@
+{ ... }:
+{
+  perSystem = { config, ... }:
+    let
+      typescriptFlake =
+        config.overlayAttrs.extras.typescriptFlake {
+          name = "lbr-plutus-typescript";
+          src = ./.;
+          dependencies = [ config.packages."lbr-prelude-typescript-tarball" ];
```

Comment:
Do you think it's better to specify tarballs in deps or pure source? If it's tarballs, then we need to expose the 'tarballing' function, but also maybe not as typescriptFlake makes it for you! Ignore me